### ISSUE TITLE
[Question] Is it possible to run parallel groups of tests serially?

### ISSUE THREAD

**Issue by Chung-Peter** [microsoft/playwright#16876]
I have one set of tests that are dependent on another set of tests to finish first (both in the same spec.ts file). Each group of tests are independent so can be run in parallel. However, when I set 'serial' mode, I cannot set any other group of tests to 'parallel' mode.

The only way I've been able to run these two sets of tests one after the other is by running two separate commands on the CLI using grep. However, the problem is that the test report from the first group of tests are lost.

If I describe each set of tests with `test.describe.parallel()` all the tests run in parallel with tests from the second test intermingled with the first. It would be preferable, at the very least, for all the tests in the first test.describe block to be first executed by the parallel workers then add in the tests from the second set of tests but the test runner seems to want to run all the tests in parallel in no particular order.

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you disagree with this closure, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.

**Comment by Xotabu4** [comment]
You can use https://www.npmjs.com/package/playwright-merge-html-reports to merge 2 reports together

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you have additional information not present in this issue that you think will help prioritizing it, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
